django/db/models/indexes.py

```diff
@@ -0,0 +1,118 @@
+from __future__ import unicode_literals
+
+import hashlib
+
+from django.core.exceptions import ValidationError
+from django.utils.encoding import force_bytes
+
+__all__ = ['Index']
+
+# The max length of the names of the indexes (restricted to 30 due to Oracle)
+MAX_NAME_LENGTH = 30
+
+
+class Index(object):
+
+    index_type = 'idx'
+
+    def __init__(self, *fields, **kwargs):
```

Comment:
We might add `type` keyword argument later if other index types prove out to be simple enough that they don't need a separate class of their own.